components/StateHandlers/InitialStateHandler.tsx

```diff
@@ -4,27 +4,16 @@ import * as Linking from "expo-linking";
 import { useCallback, useEffect, useRef } from "react";
 import { useColorScheme } from "react-native";
 
-import config from "../../config";
 import { useAppStore } from "../../data/store/appStore";
 import { useOnboardingStore } from "../../data/store/onboardingStore";
 import { useSelect } from "../../data/store/storeHelpers";
 import {
+  getSchemedURLFromUniversalURL,
```

Comment:
what is a schemed url?

Reply:
A custom scheme like converse:// and not a http link. Would you prefer another naming?